backend/core/database.py

```diff
@@ -1222,39 +1225,39 @@ def _insert_update_hijacks(self):
             self.insert_hijacks_entries.clear()
             return num_of_entries
 
-        def _retrieve_unhandled(self, amount):
-            results = []
-            query = (
-                "SELECT key, prefix, origin_as, peer_asn, as_path, service, "
-                "type, communities, timestamp FROM bgp_updates WHERE "
-                "handled = false ORDER BY timestamp DESC LIMIT(%s)"
-            )
-            with get_ro_cursor(self.ro_conn) as db_cur:
-                db_cur.execute(query, (amount,))
-                entries = db_cur.fetchall()
-
-            for entry in entries:
-                results.append(
-                    {
-                        "key": entry[0],  # key
-                        "prefix": entry[1],  # prefix
-                        "origin_as": entry[2],  # origin_as
-                        "peer_asn": entry[3],  # peer_asn
-                        "path": entry[4],  # as_path
-                        "service": entry[5],  # service
-                        "type": entry[6],  # type
-                        "communities": entry[7],  # communities
-                        "timestamp": entry[8].timestamp(),
-                    }
-                )
-            if results:
-                self.producer.publish(
-                    results,
-                    exchange=self.update_exchange,
-                    routing_key="unhandled",
-                    retry=False,
-                    priority=2,
-                )
+        # def _retrieve_unhandled(self, amount):
```

Comment:
@slowr are you sure this is not used anywhere? It is deprecated code that is not needed anymore?